README.md

```diff
@@ -112,7 +112,7 @@ helm install custom-hedera-json-rpc-relay -f path/to/values/file.yaml ./helm-cha
     --docker-password=$GH_PAT \
     --docker-email=$GH_EMAIL
     ```
-3. Deploy this helm-chart with the addtion environment/minikube.yaml file
+3. Deploy this helm-chart with the addtion [environment/minikube.yaml](environment/minikube.yaml) file
```

Comment:
```suggestion
3. Deploy this helm-chart with the additional [environment/minikube.yaml](environment/minikube.yaml) file
```

Reply:
using the commit suggested did not go too well here.  Its impossible to do that as a signed commit.  Whoops, added a commit lower to address this.